packages/react-vapor/src/components/collapsible/CollapsibleContainerConnected.tsx

```diff
@@ -72,7 +70,10 @@ export const CollapsibleContainerDisconnected: React.FunctionComponent<
                 </div>
             }
             expandedOnMount={expandedOnMount}
-            headerClasses={classNames(styles.header, expanded ? styles.expanded : styles.collapsed)}
+            headerClasses={classNames(
+                'collapsible-container header' +
+                    (expanded ? 'collapsible-container expanded' : 'collapsible-container collapsed')
+            )}

```

Comment:
```suggestion
            headerClasses={classNames(
                'collapsible-container header',
                {expanded, 'collapsed': !expanded}                  
            )}
```

Reply:
Does this work?

Reply:
![image](https://user-images.githubusercontent.com/63734941/112888651-a149c480-90a2-11eb-956f-166ce444f045.png)
it's breaking the font color :eyes:

Reply:
In the demo link, the class `header` and `collapsible-container` are stick together which is not good
![image](https://user-images.githubusercontent.com/52677246/112889511-b83ce680-90a3-11eb-9eee-332a8a70c6e0.png)

If you separate these 2 classes, you could find that the text color white comes from `header` which needs to be fixed in the scope of `collapsible-container`
![image](https://user-images.githubusercontent.com/52677246/112889662-ed493900-90a3-11eb-9156-f359cce81037.png)

Reply:
oohh I see :eyes: